xlsx2reqif.py

```diff
@@ -114,7 +114,7 @@ def get_images(images, row, col):
         # do images:
         pictures = get_images(images, row_nr - 1, columns.index(col))
         if type(xls_req[col]) == str:
-            xls_req[col] = xls_req[col].replace("<", "&gt;")
+            xls_req[col] = xls_req[col].replace(">", "&gt;")
             xls_req[col].replace("<", "&lt;")
```

Comment:
should line 118 read
`
xls_req[col] = xls_req[col].replace("<", "&lt;")
`
?

Reply:
Good point. I did not verify if it is indeed replacing the "<". I just noticed that ">" did get replaced, but incorrectly. So that is what I fixed.